### ISSUE TITLE
[feature] wildcard per type

### ISSUE THREAD

**Issue by syffs** [prisma-labs/graphql-middleware#277]
Unless I've missed something, I don't think it's supported:
It would be great to handle wildcard middlewares as in [graphql-shield](https://github.com/maticzav/graphql-shield#per-type-wildcard-rule).

**Comment by maticzav** [comment]
@syffs, I agree! I'll try to find some time to implement this. It worked great so far with `graphql-shield`. If you find some time, I'd love to merge a PR!

**Comment by schester44** [comment]
+1 on this

**Comment by bombillazo** [comment]
Any update on this feature? It's a time saver using Shield, would be great to have for custom middleware.
